modules/end-user-guide/pages/optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc

```diff
@@ -1,6 +1,6 @@
 :_content-type: CONCEPT
 :description: Optional parameters for the URLs for starting a new workspace
-:keywords: url, urls, parameter, parameters, starting, start, workspace, workspaces, url-parameter, url-parameters, multiple-url-parameters, combine-url-parameters, several-url-parameters, add-url-parameters, concatenate, concatenate-multiple-URL-parameters, concatenate-URL-parameters, concatenating, concatenating-multiple-URL-parameters, concatenating-URL-parameters, concatenation, start-duplicate-workspaces, start-duplicated-workspaces, duplicate-workspace, duplicate-workspaces, duplicated-workspace, duplicated-workspaces, how-to-start-a-duplicate, how-to-start-duplicate, devfile, override-devfile-file-name, override-file-name, file-name, devfile-name, different-devfile-name, file-path, devfile-path, devfile-file-path, devfile-path-parameter, devfile-file-path-parameterdevfile, file-path, devfile-path, devfile-file-path, devfile-path-parameter, devfile-file-path-parameter
+:keywords: url, urls, parameter, parameters, starting, start, workspace, workspaces, url-parameter, url-parameters
```

Comment:
I'm sorry, we misunderstood each other :) @max-cx, I meant to leave out these generic keywords (and others), not keep them.

Reply:
Nah, this time you're looking at a different file (that i seem to have edited before your review): https://github.com/eclipse-che/che-docs/blob/431c1ff0d663a8d82e0ceb565e9c31f6b06f24af/modules/end-user-guide/pages/url-parameter-for-the-workspace-ide.adoc Let me have a second look at this list though.

Reply:
WDYT?
```
:keywords: concatenate, concatenation, ide-parameter, start-duplicate-workspaces, devfile-parameter, devfile-name, devfile-path
```